src/quest_manager/models.py

```diff
@@ -37,6 +37,15 @@ class Meta:
     def __str__(self):
         return self.title
 
+    def get_absolute_url(self):
```

Comment:
All new methods should have tests.  These ones should be easy, you should be able to almost directly copy the same method tests for badges, or quests.... or some other model that has these exact same methods.

Reply:
Test added. There isn't currently a test for the get_icon_url() method on any relevant model (quests, badges) so the methodology is a little unclear to me. The test objects don't generate a custom icon upon creation, so any time the method is called in a test, it returns the default url, which can't be found using (self.client.get(self.category.get_icon_url(), follow=True).